Bookshelf/Models/DbBookStore.cs

```diff
@@ -43,14 +43,14 @@ public async Task CreateAsync(Book book)
 
         public Task DeleteAsync(string id)
         {
-            Book book = _dbcontext.Books.Local.First(b => b.Id == id) 
+            Book book = _dbcontext.Books.Local.First(b => b.Id == id)

```

Comment:
It should be Single, not First as you expect only one entity to match. 
Likely both will yield the same result in best case scenario but if there are two or more, you likely want this to fail with exception as opposed to still work.